config/initializers/gds-sso.rb

```diff
@@ -2,6 +2,6 @@
   config.user_model   = "User"
   config.oauth_id     = ENV["OAUTH_ID"] || "abcdefghjasndjkasndassetmanager"
   config.oauth_secret = ENV["OAUTH_SECRET"] || "secret"
-  config.oauth_root_url = Plek.current.find("signon")
+  config.oauth_root_url = Plek.current.external_url_for("signon")
```

Comment:
Minor: I think `Plek.current` is "sort of" deprecated - see [this comment](https://github.com/alphagov/plek/blob/3b838fd022a5890946a3a3ce7e774d758e0c6320/lib/plek.rb#L148-L152). I think approved way is `Plek.new.external_url_for("signon")`. It might be nice to make this change while we're at it.

Reply:
Good spot. I've updated this to use `Plek.new` in a new commit in this branch.